main/logkeeper.go

```diff
@@ -29,15 +30,27 @@ func main() {
 	logPath := flag.String("logpath", "logkeeperapp.log", "path to log file")
 	maxRequestSize := flag.Int("maxRequestSize", 1024*1024*32,
 		"maximum size for a request in bytes, defaults to 32 MB (in bytes)")
-	traceCollectorEndpoint := flag.String("traceCollectorEndpoint", "", "OTEL Collector URL")
-	useInsecure := flag.Bool("useInsecure", false, "Use Internal DNS")
-	sampleRatio := flag.Float64("sampleRatio", 1.0, "Otel Trace Sample ratio")
 	_ = flag.String("dbhost", "", "LEGACY: this option is ignored")
 	flag.Parse()
 
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	var (
+		err                    error
+		traceCollectorEndpoint = ""
+		sampleRatio            = 1.0
+	)
+
+	if value, ok := os.LookupEnv("LK_COLLECTOR_ENDPOINT"); ok {
+		traceCollectorEndpoint = value
+	}
+	if value, ok := os.LookupEnv("LK_TRACE_RATIO"); ok {
+		if sampleRatio, err = strconv.ParseFloat(value, 64); err != nil {
```

Comment:
I'm not sure what `ParseFloat` will return in the case of error. It seems like it's probably going to set `sampleRatio` to 0 if there's an error.

Reply:
yeah the default is zero on error